onnxruntime/python/onnxruntime_pybind_state.cc

```diff
@@ -593,7 +594,7 @@ std::unique_ptr<IExecutionProvider> CreateExecutionProviderInstance(
 #endif
   } else if (type == kOpenVINOExecutionProvider) {
 #ifdef USE_OPENVINO
-    OrtOpenVINOProviderOptions params;
+    OrtOpenVINOProviderOptionsV2 params;
```

Comment:
Will changing the structure here impact backward compatibility for example if you remove vpu_fast_compile